### PR TITLE
[4.2] Empty folder

### PR DESCRIPTION
**Pull request by brianteeman**
Removes the empty folder libraries/vendor/symfony/string/Resources/bin from the distributed release.

Pull Request for Issue #38515

Test by code review and by checking that the folder doesnt exist in the pre-built package for this pr

### CODE REVIEW

**Review by zero-24 — APPROVED** [review]
Looks good to me

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on 5d962b21b23c6c7784ec319b726db153d01fccd0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38516">issues.joomla.org/tracker/joomla-cms/38516</a>.</sub>

**Comment by ReLater** [comment]
I have tested this item :white_check_mark: successfully on 5d962b21b23c6c7784ec319b726db153d01fccd0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38516">issues.joomla.org/tracker/joomla-cms/38516</a>.</sub>

**Comment by zero-24** [comment]
@brianteeman can you please add the deletion of that folder into the script.php too so this will be deleted for existing installed sites.

File: build/build.php [hunk 1]
```diff
@@ -168,6 +168,7 @@ function clean_checkout(string $dir)
 
     // symfony/*
     system('rm -rf libraries/vendor/symfony/*/Resources/doc');
+    system('rm -rf libraries/vendor/symfony/string/Resources/bin');
```
Comment (richard67) [inline build/build.php]: When alpha ordering it should come at the bottom because "*" comes before "c" comes before "s".

**Comment by richard67** [comment]
> @brianteeman can you please add the deletion of that folder into the script.php too so this will be deleted for existing installed sites.

@zero-24 As far as I remember we once have decided that we don't leave the deleted files and folders in script.php up to the PR authors in J4 but we (maintainers, mostly me) do it for them.

**Comment by brianteeman** [comment]
Thats why I didnt do it 

But to make life easier I just did it

**Comment by brianteeman** [comment]
@richard67 ps you probably need to add hotkeys.js as a folder to remove as it was present in beta releases but is now renamed as hotkeysjs

**Comment by zero-24** [comment]
> > @brianteeman can you please add the deletion of that folder into the script.php too so this will be deleted for existing installed sites.
> 
> @zero-24 As far as I remember we once have decided that we don't leave the deleted files and folders in script.php up to the PR authors in J4 but we (maintainers, mostly me) do it for them.

Ok didnt know about this change can you do a PR against the checklist so its not forgotten when other people are doing the releases?

File: administrator/components/com_admin/script.php [hunk 2]
```diff
@@ -7802,6 +7802,8 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             // From 4.2.0-beta2 to 4.2.0-beta3
             '/plugins/system/webauthn/src/Helper',
             '/plugins/system/webauthn/src/Exception',
+            // From 4.2.0-beta3 to 4.2.0
```
Comment (richard67) [inline administrator/components/com_admin/script.php]:
```suggestion
            // From 4.2.0 to 4.2.1
```
The folder is removed after 4.2.0 stable (even if removed from the downloads meanwhile) and before the upcoming 4.2.1. So I suggest to change the comment, but I do not insist on it.

**Comment by richard67** [comment]
> @richard67 ps you probably need to add hotkeys.js as a folder to remove as it was present in beta releases but is now renamed as hotkeysjs

@brianteeman Thanks, will keep that in mind.

**Review by richard67 — APPROVED** [review]

**Comment by ReLater** [comment]
I have tested this item :white_check_mark: successfully on 32f4961666d4c2006797b1fc6636e564fdcff6a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38516">issues.joomla.org/tracker/joomla-cms/38516</a>.</sub>

**Review by zero-24 — APPROVED** [review]

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 32f4961666d4c2006797b1fc6636e564fdcff6a6<br /><br />Checked the packages built by drone for this PR and checked that updating works with the update package and the changed script.php.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38516">issues.joomla.org/tracker/joomla-cms/38516</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38516">issues.joomla.org/tracker/joomla-cms/38516</a>.</sub>

**Comment by fancyFranci** [comment]
Thank you! One more step for a better release

**Pull request merged**